### PR TITLE
docs(readme): enrich readme

### LINKED ISSUE #2: Update README
The current README is a brief description of what the command-line tool does. It needs to be updated with a bit more details so that the users are aware of what the tool does. Apart from that, a GIF should also be added, which demonstrates how the CLI works and how results are displayed.

### PR DESCRIPTION
**Pull request by ruddha2001**
## Features
- Add flags
- Add example GIF
- Enrich content

## Breaking Changes
- None

## Issues
- Resolve #2

### CODE REVIEW

**Comment by ruddha2001** [comment]
@dmahajan980 Will this be okay?

**Comment by dmahajan980** [comment]
Hi @ruddha2001, thanks for your contribution. I'm running on a tight schedule at the moment, so I couldn't review your pull request for a while. But, be assured! I'll make sure this gets counted for and before Hacktoberfest ends :)

**Comment by ruddha2001** [comment]
Hello @dmahajan980!
I am so sorry to disturb you.
It is just that it is my first Hacktoberfest fest and my review period of this PR ends tomorrow. This is my last PR which needs to be accepted.
I will be so indebted to you if you can review my PR.
Once again, apologies for disturbig you.

File: README.md
Comment (dmahajan980) [inline README.md]: Seems like you made a typo. You should remove the quote.

File: README.md
Comment (dmahajan980) [inline README.md]: I think you should change this to something like, "Getting started with the CoVID-19 CLI is very simple. Kindly follow the steps given below"

File: README.md
Comment (dmahajan980) [inline README.md:18]: I would suggest not to change this step. That would be simple for the viewer.

Comment (dmahajan980) [inline README.md]: I'd suggest removing the "choose to" part from the statement.

File: README.md
Comment (dmahajan980) [inline README.md:19]: "(Use the following command to clone the repository)" sounds better IMO.

File: README.md
Comment (dmahajan980) [reply README.md]: Also, you can modify the part of your statement as "download or clone the repository (see below)." and remove the next line from the step.

File: README.md
Comment (dmahajan980) [inline README.md]: This might be a personal preference, but the previous version of the step sounds better. I'd suggest using the old wording/statement here.

File: README.md
Comment (dmahajan980) [inline README.md]: "Add a symbolic link to the global folder." would be nicer.

File: README.md
Comment (dmahajan980) [inline README.md]: I'd suggest restructuring the "Windows" and  "macOS/Linux" as two sub-headings (using two sub-bullet points) and moving the contents/steps under those sub-headings.

File: README.md
Comment (dmahajan980) [inline README.md]: Kindly add backticks to the keyword as well. The bold asterisks are fine.

File: README.md
Comment (dmahajan980) [inline README.md]: "The following flag is supported." would be better.

File: README.md
Comment (dmahajan980) [inline README.md]: You can use `th` instead of `td` for all the column headings so that the word is displayed as a heading. Don't forget to "capitalize" the headings as well.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment on putting backticks here as mentioned above.

File: README.md
Comment (dmahajan980) [inline README.md]: "The following flag is supported." would be better.

File: README.md
Comment (dmahajan980) [inline README.md]: It's fine to keep blank lines between two separate `tr` tags. But you can avoid the unnecessary blank line inside the `tr` tags. Kindly remove this and all the blank lines inside the `tr` tag.

File: README.md
Comment (dmahajan980) [inline README.md]: Kindly indent the child tags as needed (tab space of four whitespace characters). This avoids confusion for other maintainers and developers.

File: README.md
Comment (dmahajan980) [inline README.md]: Single whitespace between the point and the first letter of the point is fine. Kindly remove the extra whitespace here.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about extra whitespace here.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about extra whitespace here.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about extra whitespace here. Kindly change this at all the places with extra whitespace.

File: README.md
Comment (dmahajan980) [inline README.md]: You can get rid of the "OR" keyword. It's fine not to mention that, since you've already specified both as aliases above.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about indentation of child tags here and for all other child tags in this table.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about using `th` instead of `td` for headings and "capitalizing" them.

Comment (dmahajan980) [inline README.md]: You should probably remove the `width` from the `style` attribute for the first column since this doesn't fit in the whole command (see the screenshot below).



In general, a better way to avoid these scenarios is to use the [markdown table syntax](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/organizing-information-with-tables) instead of the HTML tags, especially when you want to use a table in "README" or some other markdown file.

File: README.md
Comment (dmahajan980) [inline README.md:60]: Same comment about removing blank lines inside the `tr` tags.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about removing "OR" here.

File: README.md
Comment (dmahajan980) [inline README.md]: It seems that you forgot to commit the GIF file. Kindly add that as well. I'd suggest making an `assets` folder in the repository and adding your GIF there so that you can use it.

File: README.md
Comment (dmahajan980) [inline README.md]: The `br` tag creates too much of whitespace below the table. It's fine to not use it and below the other tables that you created.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about removing `br` tag here. A simple blank line would be fine here.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about `br` tag as above.

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about `br` tags as mentioned above.

File: README.md
Comment (dmahajan980) [inline README.md]: The "countryName" should be in backticks so that readers don't oversee it as a regular wording.

File: README.md
Comment (dmahajan980) [inline README.md]: It's fine to join this statement with the above statement. You can remove the `br` tag above this, as I mentioned earlier.

Comment (dmahajan980) [reply README.md]: Don't forget to add a "period" at the end of this statement.

File: README.md
Comment (dmahajan980) [inline README.md]: Although this would work, you should probably capitalize the country name, i.e., "India".

**Comment by ruddha2001** [comment]
@dmahajan980 Thank you so much for taking your time out!
I have worked on the comments as you said.

Also, since this is my first Hacktoberfest I am a bit confused. Should not this repo have the word "hacktoberfest" as a topic as per the new rules?

Anyways, thank you so much for your time!

File: README.md
Comment (dmahajan980) [inline README.md]: I think you missed the point of having a second example here. You should change this to the alias flag while keeping the example same as above. So, this should probably be `covid19 stats -c India` or some other country name that you use in the above example.

**Review by dmahajan980 — CHANGES_REQUESTED** [review]

**Comment by dmahajan980** [comment]
@ruddha2001 apologies for the delay in reviewing your pull request. Although the README does look more structured than before, I've suggested some changes to make it even better. Kindly go through them and let me know if you have any doubts or something's unclear. Once you submit the suggested changes, we can proceed further and merge the pull if everything seems fine :)

**Comment by dmahajan980** [comment]
> Also, since this is my first Hacktoberfest I am a bit confused. Should not this repo have the word "hacktoberfest" as a topic as per the new rules?

@ruddha2001 Thanks for reminding me about the change in rules this year. I'll go through the new rules and make relevant changes as needed before we hop on to merging your pull request.

File: README.md
Comment (ruddha2001) [reply README.md]: The GIF is committed and appears in my GitHub tree as well.
However, I will change `readme-assets` to `assets`.

File: README.md
Comment (ruddha2001) [reply README.md]: This double whitespace is being added by `Prettier`.
Should I manually remove them then?

File: README.md
Comment (ruddha2001) [reply README.md]: Thank you for this. I was not well versed with markdown.
But I will definitely use this syntax for this project and all my future projects.

**Comment by ruddha2001** [comment]
@dmahajan980 Thank you so much for your comments. I really learnt a lot today.
So I made the following changes:
* Used `Prettier` so that all identations are consistent.
* Used markdown syntax for tables as well.
* Worked on the other comments as asked.

File: README.md
Comment (dmahajan980) [reply README.md]: Yes, please. That'd be fine, given that you used Prettier. Kindly make the changes at all the places that have double whitespaces.

File: README.md
Comment (dmahajan980) [inline README.md]: Seems that you intended to break the line, but the markdown actually renders the two continuous lines as a single line. For breaking the lines, simply add two whitespace characters at the end of this line and at other places where you wanted to break the line.

Comment (dmahajan980) [reply README.md]: I'd suggest making this bold so that it seems like a title.

File: README.md
Comment (dmahajan980) [inline README.md]: I'd suggest using "Open `cmd.exe` as Administrator and run the following command inside the root of the project directory"

File: README.md
Comment (dmahajan980) [inline README.md]: "Open the terminal and run the following command inside the root of the project directory" would be better.

File: README.md
Comment (dmahajan980) [inline README.md]: Seems like you made a typing error. It should be `--version` instead.

File: README.md
Comment (dmahajan980) [inline README.md]: It should be `--country` instead.

**Review by dmahajan980 — CHANGES_REQUESTED** [review]

File: README.md
Comment (dmahajan980) [inline README.md]: Sorry for asking this again, but I feel that it'd be better if we set this as a standard subheading instead of a bullet point. So, in that case, this line would be

```suggestion
### `covid19`  
```

File: README.md
Comment (dmahajan980) [inline README.md]: Same comment about changing this to a subheading.

```suggestion
### `covid19 stats` 
```

**Review by dmahajan980 — CHANGES_REQUESTED** [review]

File: README.md
Comment (dmahajan980) [inline README.md]: You don't need to add a trailing comma. A period is fine IMO.

File: README.md
Comment (dmahajan980) [reply README.md]: Also, don't forget to remove the indentation (due to the bullet point) of the contents under the subsection.

**Comment by dmahajan980** [comment]
@ruddha2001 I think we are close enough. Once you update the pull request, I can give it a final review, and hopefully, we would be good to merge this.

**Comment by ruddha2001** [comment]
@dmahajan980 Apologies for the delay in the commit.
This one has to be the most hectic yet the most fruitful PR I am doing this Hacktoberfest.
Thank you so much for being so patient and helping me out with this.

File: README.md
Comment (dmahajan980) [inline README.md]: Seems like you haven't removed the extra whitespace between the point number and the first alphabetic character. Kindly complete this change at all the places based on our previous discussion.

File: README.md
Comment (dmahajan980) [inline README.md]: You need to indent these bullet points so that it comes directly under the third point (given above).

File: README.md
Comment (dmahajan980) [inline README.md]: This should be "macOS/Linux".

File: README.md
Comment (dmahajan980) [inline README.md]: This should be "CoVID-19".

File: README.md
Comment (dmahajan980) [inline README.md]: As I suggested earlier, you probably should use the same country name in both commands. This reduces the chance of confusion with the two alias flags. So you should set both to either "US" or "India".

File: README.md
Comment (dmahajan980) [inline README.md:30]: I liked your version of the sentence here, i.e., "Add a symbolic link to the global folder" . I meant to restore the second point only. I'll be happy to have your version of the statement here.

Comment (dmahajan980) [inline README.md]: "Add a symbolic link to the global folder"

Re-commenting this so that you don't miss it again :)

File: README.md
Comment (ruddha2001) [reply README.md]: But this is the default behaviour of `Prettier`.
Should I still change this?

File: README.md
Comment (ruddha2001) [reply README.md]: Alright. I finally get what is to be done.
I have now made them sub-bullets under Point 3.

**Comment by dmahajan980** [comment]
@ruddha2001 I've re-commented the points and changes that you missed in the previous reviews. I am very positive that these should be the final changes. I know PR reviews might become hectic and overly long sometimes, but I am happy that you worked on this patiently :)

Once you've completed all the final changes, we should proceed to merging this pull request and marking this as a "Hacktoberfest-accepted" pull.

**Comment by ruddha2001** [comment]
@dmahajan980 I have made all the changes as listed.
Except for the whitespaces because `Prettier` keeps on adding those every single time. Should I manually remove them?

File: README.md
Comment (dmahajan980) [reply README.md]: Yes. It is recommended to remove the unnecessary things from the project files, which in this case is this extra whitespace. Having used "Prettier" in the past, I can recall that it does provide an option to change the configuration. However, for this minor change, you can do it manually and later work on your local configuration to avoid the extra whitespace in your future projects.

**Comment by dmahajan980** [comment]
@ruddha2001 as I said earlier, you probably should remove it manually for now. You can later work on your "Prettier" configuration to avoid extra whitespace in the future.

**Comment by ruddha2001** [comment]
@dmahajan980 I used `Prettier` configuration itself. That is something new I learnt!
Hopefully, all changes are covered now.

**Comment by dmahajan980** [comment]
@ruddha2001 that looks great 🔥 
Thanks for being patient and contributing to this repository. The CLI will be published on NPM in a few days and this pull request will definitely help the new users in using the CLI. I wish you all the best for Hacktoberfest and I hope you continue to contribute to open-source :)

**Pull request merged**
